Code/oncareer/src/components/LandingPage/Screenshot/Screenshot.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import pic from './Dashboard.svg';
+import classes from './Screenshot.css';
+
+const Screenshot = (props) => (
+    <div className={classes.screenshot} style={{height: props.height}}>
+        <img src={pic} width='100%' height='100%' />
```

Comment:
I think you may need to include an alt property in img tag. Probably appears as a warning in React console.